custom/icds_reports/data_pull/awc_phone_numbers.py

```diff
@@ -0,0 +1,56 @@
+# get phone numbers of AWCs of specific states
+
+import csv
+from corehq.apps.app_manager.const import USERCASE_TYPE
+from corehq.apps.locations.models import SQLLocation
+from corehq.apps.locations.dbaccessors import get_user_ids_by_location
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+from django.db.models import Q
+from corehq.apps.es.cases import CaseES
+from corehq.elastic import ES_EXPORT_INSTANCE
+
+domain = "icds-cas"
+real_state_names = [
+    loc.name 
+    for loc in SQLLocation.active_objects.filter(domain=domain, location_type__code='state') 
+    if loc.metadata.get('is_test_location') != 'test'
+]
+
+case_accessor = CaseAccessors(domain)
+multiple_usercases_found_for_user_ids = []
+no_usercases_found_for_user_ids = []
+
+q = CaseES(es_instance_alias=ES_EXPORT_INSTANCE).domain(domain).term('type', USERCASE_TYPE)
```

Comment:
@esoergel just wanted to get your input on this.
there are two filters i considered adding to this. one was `closed` and other was `deleted`
when i added `opened` i got results but as soon as i added `deleted` filter i got no hits
this is how i added it
```
q = (CaseES(es_instance_alias=ES_EXPORT_INSTANCE).
     domain(domain).
     term('type', USERCASE_TYPE).
     term('deleted', False).
     term('closed', False)
     )
```

Reply:
`deleted` isn't a field in the [case ES mapping](https://github.com/dimagi/commcare-hq/blob/5d6be3a9d2568714a58445667267e11f563f7120/corehq/pillows/mappings/case_mapping.py), so you can't search by it.